Cargo.toml

```diff
@@ -170,7 +170,7 @@ thiserror = "1.0"
 tikv-jemallocator = "0.5"
 tikv-jemalloc-ctl = "0.5"
 tokio = { version = "1.17", features = ["macros"] }
-tokio-epoll-uring = { git = "https://github.com/neondatabase/tokio-epoll-uring.git" , branch = "main" }
```

Comment:
We need to change this back once tokio-epoll-uring changes are on main.

Reply:
Let's just merge the tokio-epoll-uring change first